popup.js

```diff
@@ -85,6 +88,7 @@ function onFontsLoad(json, fromCache) {
     lscache.set(LS_FONTS_API, json, 60*12);
   }
   loadVisibleFonts();
```

Comment:
I think loadVisibleFonts should get called after onsubsetchange/onsearchchange so that it doesn't unnecessary load fonts that are about to get hidden.

Reply:
Something wasn't working correctly when I first tried to put them in the reverse order but you're right, it doesn't make sense to be calling `loadVisibleFonts()` so much. I'll look into it.